azkustodata/kusto.go

```diff
@@ -172,6 +177,86 @@ func (c *Client) Query(ctx context.Context, db string, query Statement, options
 	return iter, nil
 }
 
+func (c *Client) newV1(
```

Comment:
New is usually reserved for instance creation.
Why do you use it in query/command APIs?

The set of API names need working.
Possible [Mgmt, QueryV1, QueryV2, IterativeQuery] where Iterative is V2 implicitly.

Reply:
Again, the conflict with the old names